declarative-examples/simple-examples/postConditionOrder.groovy

```diff
@@ -0,0 +1,28 @@
+pipeline {
+    // no agent required to run here. All steps run in flyweight executor on Master
+    agent none
+
+    stages {
+        stage("foo") {
+            steps {
+                echo "hello"
+            }
+        }
+    }
+    post {
+    // these steps will run at the end of the pipeline based on the condition.
+    // The conditions run in this order regardless of their arrangement
```

Comment:
Not sure this isn't ambiguous.

Perhaps: *These blocks are always evaluated in the order below, i.e. `always` first, then `changed`, etc., regardless of the order they appear in the Jenkinsfile.*